src/OpenTelemetry.Instrumentation.GrpcNetClient/OpenTelemetry.Instrumentation.GrpcNetClient.csproj

```
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(TargetFrameworksForGrpcNetClientInstrumentation)</TargetFrameworks>
     <Description>gRPC for .NET client instrumentation for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.Grpc-</MinVerTagPrefix>
```

Match it with the project name?

```suggestion
    <MinVerTagPrefix>Instrumentation.GrpcNetClient-</MinVerTagPrefix>
```